### PR TITLE
Support vcs roots

### PR DESCRIPTION
**Pull request by lukaszwawrzyk**

### CODE REVIEW

File: driver/bindings/junit/src/main/scala/org/virtuslab/ideprobe/DSL.scala [hunk 1]
```diff
@@ -13,19 +13,19 @@ final class RunningIntelliJFixture(
...
+  def runShell(args: Seq[String]): CommandResult = Shell.run(path, args: _*)
 
   def config: Config = fixture.config
 
   def intelliJPaths: IntelliJPaths = installedIntelliJ.paths
 
-  def run[A](action: RunningIntelliJFixture => A): A = {
-    val running = fixture.startIntelliJ(workspace, installedIntelliJ)
-    val data = new RunningIntelliJFixture(workspace, running.probe, config, intelliJPaths)
+  def runIntellij[A](action: RunningIntelliJFixture => A): A = {
```
Comment (marek1840) [inline driver/bindings/junit/src/main/scala/org/virtuslab/ideprobe/DSL.scala:26]: maybe `runIDE`? it's shorter and more generic

File: driver/sources/src/main/scala/org/virtuslab/ideprobe/dependencies/IntelliJVersion.scala [hunk 2]
```diff
@@ -3,5 +3,5 @@ package org.virtuslab.ideprobe.dependencies
 final case class IntelliJVersion(build: String) extends AnyVal
 
 object IntelliJVersion {
-  val Latest = IntelliJVersion("202.5792.28-EAP-SNAPSHOT")
+  val Latest = IntelliJVersion("202.6250.13-EAP-SNAPSHOT")
```
Comment (marek1840) [inline driver/sources/src/main/scala/org/virtuslab/ideprobe/dependencies/IntelliJVersion.scala:6]: what about the version of the plugin itself in `build.sbt`? Should it stay unchanged?

File: driver/bindings/junit/src/main/scala/org/virtuslab/ideprobe/DSL.scala (hunk 1 above)
Comment (lukaszwawrzyk) [reply driver/bindings/junit/src/main/scala/org/virtuslab/ideprobe/DSL.scala:26]: I'd keep intellij for now. We have `Intellij` all over the code/api.

File: driver/sources/src/main/scala/org/virtuslab/ideprobe/dependencies/IntelliJVersion.scala (hunk 2 above)
Comment (lukaszwawrzyk) [reply driver/sources/src/main/scala/org/virtuslab/ideprobe/dependencies/IntelliJVersion.scala]: I used buildinfo to have it in one place

**Pull request merged**